gpx.js

```diff
@@ -445,6 +445,8 @@ L.GPX = L.FeatureGroup.extend({
       _ = el[i].getElementsByTagName('ele');
       if (_.length > 0) {
         ll.meta.ele = parseFloat(_[0].textContent);
+      } else {
+        ll.meta.time = new Date('1970-01-01T00:00:00');
```

Comment:
Are you sure this isn't supposed to be in the block above, line 443?

Reply:
Sorry. Of course you are right.